.github/workflows/rust.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        rustc: [1.67.1, stable] # MSVR and current stable rustc
+        rustc: [1.70.0, stable] # MSVR and current stable rustc
```

Comment:
I don't think the version bump is necessary if it is just for `println!("{stuff}");`. at least this makes the GitHub CI check unable to pass. Please keep the minimal at 1.67.1 and do the bump in a separate PR.

Reply:
> I don't think the version bump is necessary if it is just for `println!("{stuff}");`. at least this makes the GitHub CI check unable to pass. Please keep the minimal at 1.67.1 and do the bump in a separate PR.

@kennytm `println!("{stuff}");` was stabilized in [Rust 1.58.0](https://blog.rust-lang.org/2022/01/13/Rust-1.58.0/#captured-identifiers-in-format-strings), so it's also available in MSRV 1.67.1.

The reason for the MSRV change is that the MSRV of the `image` crate was changed to 1.70.0 since version 0.25.5.[^1] So if you run the CI now with c578416 it will fail.